### DISCUSSION TITLE
Misorientation axis of 2 phases

### DISCUSSION THREAD

**Discussion by d-Wang8** [mtex-toolbox/mtex#1429] [Ask Anything]
EBSD software only gave me the smallest misorientations between 2 phases. But it won't give me the miller indices of the rotational axis; so I wrote the following scripts for 2 phases (85 and tau):
```
cs85 = crystalSymmetry('R3m',[12.667,12.667,7.942],[90*degree 90*degree 120*degree]);% Al8Mn5
csTau = crystalSymmetry('P4/mmm',[2.77,2.77,3.54],[90*degree,90*degree,90*degree]); %Tau
o1 = orientation.byEuler([197.5,1.2,181.3]*degree,csTau);
o2 = orientation.byEuler([188,84.5,197.2]*degree,cs85);
%%
 mori = inv(o1) * o2;

 t85 = axis(inv(o1)*o2,cs85)
 tTau = axis(inv(o1)*o2,csTau)
 mori.angle/degree
 round(t85)
 round(Miller(t85,cs85,'UVTW'))
 round(tTau)
 round(Miller(tTau,csTau,'uvw'))
```
However, this script only gives me some weird axis like [-12,3,1] of tau phase and smallest misorientation angle, which might not be useful to study the OR between these 2 phases. So I am wondering is it possible in we can disregard the symmetry of one phase, and thereby ask MTEX to give me a series of angles and corresponding  rotational axis? like the MTEX scripts used for Misorientation and twinning analysis : angle(ori1, ori2.symmetrise,'noSymmetry')./ degree

thanks 
Di

**Comment by kilir** [comment]
Hi,
does the following help?
```
mori = inv(o1) * o2;
moriS = symmetrise(mori)

axis(moriS)
axis(moriS,cs85)
axis(moriS,csTau)
angle(moriS)
```
Cheers,
Rüdiger
